Lombiq.Tests.UI/Services/UITestExecutionSession.cs

```diff
@@ -61,6 +61,12 @@ public UITestExecutionSession(
 
     public async Task<bool> ExecuteAsync(int retryCount, string dumpRootPath)
     {
+        _configuration.TestCancellationToken.Register(() =>
```

Comment:
I'm not sure, but I think `System.Threading.CancellationTokenRegistration` returned by `CancellationToken.Register` needs to be disposed.

Testing `_configuration.TestCancellationToken.IsCancellationRequested` also wellcome before starting the whole ceremony.

Reply:
It indeed needs to be disposed, so I added that.

I don't really understand the rest of your comment: xUnit 3 supports cancellations. If we want the tests to clean up after themselves when such a cancellation happens, hooking into it is necessary. Running `ShutdownAsync()` will do the clean-up. Thus after this, a cancelled test won't leave ChromeDriver processes or browser windows open.